### ISSUE TITLE
App hangs on v4.8.0 due to Xlib deps IME

### ISSUE THREAD

**Issue by mingtsay** [telegramdesktop/tdesktop#26228]
### Steps to reproduce

1. Download Telegram Desktop `v4.8.0`, or `v4.8.1`  Linux 64 bit: Binary from Releases.
2. Unzip binaries `Telegram` and `Updater` using `tar` or `file-roller` to `~/app/Telegram`
3. Double click `Telegram` or run command `./Telegram` via terminal.
4. It hangs and do nothing when you input by keyboard and/or mouse.

### Expected behaviour

It should load messages. It should react when mouse/keyboard input.

### Actual behaviour

It hangs and do nothing when you input by keyboard and/or mouse.

### Operating system

Debian GNU/Linux 11 (bullseye) / Linux 5.10.0-21-amd64 / Cinnamon 4.8.6-2+deb11u1 / LightDM 1.26.0-7

### Version of Telegram Desktop

v4.8.0, v4.8.1

### Installation source

Static binary from official website

### Logs

```shell
[2023.04.26 00:00:24] Launched version: 4008001, install beta: [FALSE], alpha: 0, debug mode: [FALSE]
[2023.04.26 00:00:24] Executable dir: /home/mt/apps/Telegram/, name: Telegram
[2023.04.26 00:00:24] Initial working dir: /media/sda/mt/.local/share/TelegramDesktop/
[2023.04.26 00:00:24] Working dir: /home/mt/.local/share/TelegramDesktop/
[2023.04.26 00:00:24] Command line: /home/mt/apps/Telegram/Telegram -noupdate
[2023.04.26 00:00:24] Executable path before check: /home/mt/apps/Telegram/Telegram
[2023.04.26 00:00:24] Logs started
[2023.04.26 00:00:24] Launcher filename: org.telegram.desktop._89a1235dd57ed936ddbd7811910d5b3d.desktop
[2023.04.26 00:00:24] Connecting local socket to 358046203513b99f87535da14567e73a-{87A94AB0-E370-4cde-98D3-ACC110C5967D}...
[2023.04.26 00:00:24] Socket connect error 0, starting server and app...
[2023.04.26 00:00:24] Moved logging from '/home/mt/.local/share/TelegramDesktop/log_start0.txt' to '/home/mt/.local/share/TelegramDesktop/log.txt'!
[2023.04.26 00:00:24] Global devicePixelRatio: 1
[2023.04.26 00:00:24] Primary screen DPI: 96, Base: 96.
[2023.04.26 00:00:24] Computed screen scale: 100
[2023.04.26 00:00:24] DevicePixelRatio: 1
[2023.04.26 00:00:24] ScreenScale: 100
[2023.04.26 00:00:24] System tray available: [TRUE]
[2023.04.26 00:00:24] Font: from ':/gui/fonts/DAOpenSansRegular.ttf' loaded 'DAOpenSansRegular'
[2023.04.26 00:00:24] Font: from ':/gui/fonts/DAVazirRegular.ttf' loaded 'DAVazirRegular'
[2023.04.26 00:00:24] Font: from ':/gui/fonts/DAOpenSansRegularItalic.ttf' loaded 'DAOpenSansRegularItalic'
[2023.04.26 00:00:24] Font: from ':/gui/fonts/DAVazirRegular.ttf' loaded 'DAVazirRegular'
[2023.04.26 00:00:24] Font: from ':/gui/fonts/DAOpenSansSemiboldAsBold.ttf' loaded 'DAOpenSansSemibold'
[2023.04.26 00:00:24] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2023.04.26 00:00:24] Font: from ':/gui/fonts/DAOpenSansSemiboldItalicAsBold.ttf' loaded 'DAOpenSansSemiboldItalic'
[2023.04.26 00:00:24] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2023.04.26 00:00:24] Font: from ':/gui/fonts/DAOpenSansSemiboldAsBold.ttf' loaded 'DAOpenSansSemibold'
[2023.04.26 00:00:24] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2023.04.26 00:00:24] Font: from ':/gui/fonts/DAOpenSansSemiboldItalicAsBold.ttf' loaded 'DAOpenSansSemiboldItalic'
[2023.04.26 00:00:24] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2023.04.26 00:00:24] Icon theme: mate
[2023.04.26 00:00:24] Fallback icon theme: 
[2023.04.26 00:00:24] App Info: reading settings...
[2023.04.26 00:00:24] App Info: reading encrypted settings...
[2023.04.26 00:00:24] Lang Info: Loaded cached, keys: 4996
[2023.04.26 00:00:24] OpenAL Logging Level: (not set)
[2023.04.26 00:00:24] Audio Playback Devices: Baffin HDMI/DP Audio [Radeon RX 550 640SP / RX 560/560X] Digital Stereo (HDMI 4);UAB-80 數位立體聲 (IEC958);Starship/Matisse HD Audio Controller 數位立體聲 (IEC958);內部音效 類比立體聲
[2023.04.26 00:00:24] Audio Playback Default Device: Baffin HDMI/DP Audio [Radeon RX 550 640SP / RX 560/560X] Digital Stereo (HDMI 4)
[2023.04.26 00:00:24] Audio Capture Devices: UAB-80 單聲道;Monitor of UAB-80 數位立體聲 (IEC958);Monitor of Starship/Matisse HD Audio Controller 數位立體聲 (IEC958);Monitor of 內部音效 類比立體聲;內部音效 類比立體聲;Monitor of Baffin HDMI/DP Audio [Radeon RX 550 640SP / RX 560/560X] Digital Stereo (HDMI 4)
[2023.04.26 00:00:24] Audio Capture Default Device: UAB-80 單聲道
[2023.04.26 00:00:24] App Info: reading accounts info...
[2023.04.26 00:00:24] App Info: reading encrypted info...
[2023.04.26 00:00:24] App Info: reading encrypted mtp config...
[2023.04.26 00:00:24] OpenGL Profile: Compatibility.
[2023.04.26 00:00:24] OpenGL Renderer: Radeon RX550/550 Series (POLARIS12, DRM 3.40.0, 5.10.0-21-amd64, LLVM 11.0.1)
[2023.04.26 00:00:24] OpenGL Vendor: AMD
[2023.04.26 00:00:24] OpenGL Version: 4.6 (Compatibility Profile) Mesa 20.3.5
[2023.04.26 00:00:24] OpenGL Extensions: GL_NV_blend_square, GL_ARB_texture_cube_map_array, GL_IBM_rasterpos_clip, GL_EXT_texture_shadow_lod, GL_EXT_secondary_color, GL_EXT_framebuffer_multisample, GL_ARB_shader_storage_buffer_object, GL_ARB_tessellation_shader, GL_ARB_transform_feedback3, GL_EXT_EGL_image_storage, GL_EXT_shadow_funcs, GL_NV_copy_depth_to_color, GL_EXT_texture_object, GL_ATI_draw_buffers, GL_ARB_shader_draw_parameters, GL_ARB_direct_state_access, GL_ARB_indirect_parameters, GL_AMD_framebuffer_multisample_advanced, GL_ARB_gpu_shader_int64, GL_ARB_copy_buffer, GL_ARB_half_float_vertex, GL_EXT_texture_compression_dxt1, GL_EXT_texture_shared_exponent, GL_ARB_ES3_compatibility, GL_KHR_texture_compression_astc_ldr, GL_ARB_internalformat_query, GL_EXT_vertex_array_bgra, GL_ARB_texture_query_levels, GL_ARB_shader_group_vote, GL_EXT_blend_subtract, GL_ARB_compute_variable_group_size, GL_ARB_multi_draw_indirect, GL_ARB_blend_func_extended, GL_EXT_timer_query, GL_ARB_sampler_objects, GL_EXT_stencil_two_side, GL_ARB_derivative_control, GL_ARB_parallel_shader_compile, GL_ARB_point_parameters, GL_NV_copy_image, GL_EXT_memory_object_fd, GL_KHR_no_error, GL_ARB_texture_buffer_range, GL_EXT_texture, GL_EXT_window_rectangles, GL_ARB_transform_feedback2, GL_ARB_compressed_texture_pixel_storage, GL_EXT_direct_state_access, GL_AMD_draw_buffers_blend, GL_ARB_texture_mirror_clamp_to_edge, GL_ARB_conservative_depth, GL_ARB_texture_filter_anisotropic, GL_ARB_draw_elements_base_vertex, GL_ARB_conditional_render_inverted, GL_EXT_shader_samples_identical, GL_ARB_transpose_matrix, GL_MESA_pack_invert, GL_ARB_clip_control, GL_ARB_shading_language_100, GL_AMD_pinned_memory, GL_EXT_EGL_sync, GL_INGR_blend_func_separate, GL_ATI_texture_float, GL_EXT_copy_texture, GL_ATI_fragment_shader, GL_ARB_framebuffer_sRGB, GL_ARB_draw_buffers_blend, GL_ARB_texture_buffer_object_rgb32, GL_ARB_texture_rectangle, GL_ARB_texture_buffer_object, GL_MESA_shader_integer_functions, GL_ARB_separate_shader_objects, GL_ARB_fragment_program_shadow, GL_EXT_point_parameters, GL_EXT_draw_instanced, GL_ARB_copy_image, GL_ARB_texture_barrier, GL_IBM_multimode_draw_arrays, GL_ARB_color_buffer_float, GL_ARB_texture_mirrored_repeat, GL_NV_alpha_to_coverage_dither_control, GL_ARB_vertex_attrib_64bit, GL_AMD_vertex_shader_viewport_index, GL_ARB_viewport_array, GL_ARB_get_program_binary, GL_EXT_packed_pixels, GL_INTEL_blackhole_render, GL_ARB_draw_indirect, GL_EXT_vertex_array, GL_ARB_texture_multisample, GL_ARB_texture_rg, GL_ATI_separate_stencil, GL_ARB_depth_clamp, GL_MESA_framebuffer_flip_y, GL_NV_texgen_reflection, GL_EXT_blend_minmax, GL_ARB_map_buffer_alignment, GL_ARB_shader_clock, GL_ARB_half_float_pixel, GL_ARB_vertex_buffer_object, GL_ARB_ES3_1_compatibility, GL_EXT_framebuffer_blit, GL_ARB_sync, GL_ARB_explicit_attrib_location, GL_ARB_shader_ballot, GL_ARB_vertex_type_10f_11f_11f_rev, GL_ARB_ES2_compatibility, GL_ARB_pixel_buffer_object, GL_ARB_shader_atomic_counter_ops, GL_IBM_texture_mirrored_repeat, GL_ARB_shading_language_include, GL_KHR_robust_buffer_access_behavior, GL_EXT_gpu_shader4, GL_NV_vdpau_interop, GL_ARB_texture_stencil8, GL_ARB_enhanced_layouts, GL_EXT_rescale_normal, GL_EXT_shader_image_load_formatted, GL_KHR_debug, GL_NV_texture_rectangle, GL_ARB_compatibility, GL_ARB_framebuffer_object, GL_ARB_occlusion_query2, GL_AMD_seamless_cubemap_per_texture, GL_ARB_point_sprite, GL_ARB_polygon_offset_clamp, GL_ARB_robust_buffer_access_behavior, GL_AMD_shader_stencil_export, GL_ARB_transform_feedback_overflow_query, GL_ARB_vertex_type_2_10_10_10_rev, GL_EXT_texture_sRGB_R8, GL_NV_half_float, GL_ARB_sample_shading, GL_ARB_provoking_vertex, GL_ARB_invalidate_subdata, GL_AMD_shader_trinary_minmax, GL_ARB_occlusion_query, GL_ARB_texture_view, GL_AMD_depth_clamp_separate, GL_ARB_shader_precision, GL_AMD_query_buffer_object, GL_ATI_texture_env_combine3, GL_APPLE_packed_pixels, GL_NV_texture_barrier, GL_ARB_spirv_extensions, GL_ARB_texture_compression, GL_ARB_clear_texture, GL_EXT_draw_buffers2, GL_ARB_multitexture, GL_ARB_shadow, GL_ARB_fragment_program, GL_EXT_shader_image_load_store, GL_ARB_explicit_uniform_location, GL_ARB_robustness, GL_ARB_query_buffer_object, GL_ARB_buffer_storage, GL_ARB_gpu_shader_fp64, GL_ARB_transform_feedback_instanced, GL_ARB_depth_texture, GL_ARB_draw_instanced, GL_SGIS_texture_border_clamp, GL_ARB_stencil_texturing, GL_ARB_shader_image_load_store, GL_NVX_gpu_memory_info, GL_KHR_blend_equation_advanced, GL_SGIS_texture_lod, GL_NV_light_max_exponent, GL_ARB_base_instance, GL_EXT_compiled_vertex_array, GL_ARB_vertex_program, GL_EXT_multi_draw_arrays, GL_ARB_fragment_coord_conventions, GL_AMD_texture_texture4, GL_SGIS_texture_edge_clamp, GL_ARB_arrays_of_arrays, GL_NV_conditional_render, GL_AMD_multi_draw_indirect, GL_OES_EGL_image, GL_ARB_texture_non_power_of_two, GL_ARB_fragment_layer_viewport, GL_KHR_robustness, GL_ARB_shader_image_size, GL_ARB_vertex_array_object, GL_EXT_texture_rectangle, GL_S3_s3tc, GL_ARB_shading_language_420pack, GL_ARB_texture_gather, GL_NV_primitive_restart, GL_EXT_texture_compression_latc, GL_EXT_depth_bounds_test, GL_KHR_context_flush_control, GL_MESA_texture_signed_rgba, GL_ARB_instanced_arrays, GL_KHR_parallel_shader_compile, GL_EXT_texture_integer, GL_ARB_shading_language_packing, GL_ATI_texture_compression_3dc, GL_ARB_map_buffer_range, GL_EXT_texture_sRGB, GL_ARB_compute_shader, GL_ARB_texture_cube_map, GL_EXT_polygon_offset_clamp, GL_ARB_bindless_texture, GL_ARB_internalformat_query2, GL_ARB_clear_buffer_object, GL_EXT_framebuffer_sRGB, GL_ARB_uniform_buffer_object, GL_EXT_texture_mirror_clamp, GL_ARB_seamless_cubemap_per_texture, GL_ARB_shader_viewport_layer_array, GL_ARB_texture_compression_bptc, GL_EXT_texture_edge_clamp, GL_NV_packed_depth_stencil, GL_ATI_texture_mirror_once, GL_ARB_program_interface_query, GL_EXT_vertex_attrib_64bit, GL_EXT_texture_sRGB_decode, GL_NV_fog_distance, GL_ARB_shader_bit_encoding, GL_NV_texture_env_combine4, GL_EXT_texture_cube_map, GL_EXT_pixel_buffer_object, GL_ANGLE_texture_compression_dxt5, GL_AMD_conservative_depth, GL_ARB_window_pos, GL_ARB_fragment_shader, GL_ARB_seamless_cube_map, GL_EXT_texture_array, GL_EXT_texture_swizzle, GL_EXT_texture_compression_s3tc, GL_ARB_texture_rgb10_a2ui, GL_ARB_shader_objects, GL_EXT_texture3D, GL_ARB_depth_buffer_float, GL_ARB_draw_buffers, GL_EXT_texture_buffer_object, GL_ARB_texture_border_clamp, GL_ATI_blend_equation_separate, GL_EXT_draw_range_elements, GL_ARB_gpu_shader5, GL_ARB_sparse_buffer, GL_EXT_packed_float, GL_EXT_provoking_vertex, GL_ARB_shader_stencil_export, GL_AMD_vertex_shader_layer, GL_EXT_bgra, GL_ARB_texture_env_crossbar, GL_OES_read_format, GL_ARB_texture_storage, GL_EXT_semaphore_fd, GL_ARB_vertex_attrib_binding, GL_EXT_texture_compression_rgtc, GL_EXT_fog_coord, GL_ARB_texture_env_add, GL_EXT_blend_equation_separate, GL_ARB_shader_texture_lod, GL_ARB_shader_texture_image_samples, GL_EXT_gpu_program_parameters, GL_EXT_stencil_wrap, GL_EXT_texture_filter_anisotropic, GL_ARB_texture_swizzle, GL_EXT_texture_lod_bias, GL_ARB_texture_env_dot3, GL_ARB_timer_query, GL_ARB_vertex_shader, GL_EXT_texture_env_combine, GL_ARB_texture_compression_rgtc, GL_ARB_cull_distance, GL_ARB_shader_atomic_counters, GL_NV_depth_clamp, GL_EXT_abgr, GL_EXT_separate_specular_color, GL_EXT_transform_feedback, GL_AMD_performance_monitor, GL_ARB_multi_bind, GL_KHR_texture_compression_astc_sliced_3d, GL_EXT_blend_func_separate, GL_EXT_framebuffer_multisample_blit_scaled, GL_EXT_subtexture, GL_ARB_pipeline_statistics_query, GL_SGIS_generate_mipmap, GL_EXT_packed_depth_stencil, GL_ARB_multisample, GL_MESA_window_pos, GL_ARB_texture_query_lod, GL_ARB_texture_env_combine, GL_SUN_multi_draw_arrays, GL_ARB_shader_subroutine, GL_EXT_semaphore, GL_ATI_meminfo, GL_ARB_framebuffer_no_attachments, GL_ARB_debug_output, GL_EXT_framebuffer_object, GL_ANGLE_texture_compression_dxt3, GL_ARB_texture_storage_multisample, GL_EXT_blend_color, GL_EXT_shader_integer_mix, GL_ARB_gl_spirv, GL_EXT_texture_env_add, GL_NV_shader_atomic_int64, GL_ARB_vertex_array_bgra, GL_ARB_get_texture_sub_image, GL_EXT_texture_env_dot3, GL_ARB_texture_float, GL_EXT_memory_object, GL_EXT_texture_snorm
[2023.04.26 00:00:24] OpenGL: QOpenGLContext created, version: 4.4.
[2023.04.26 00:00:25] OpenGL: [TRUE] (Window)
```

**Comment by ilya-fedin** [comment]
Does this command hang?
`busctl --user get-property org.freedesktop.portal.Desktop /org/freedesktop/portal/desktop org.freedesktop.portal.FileChooser version`

**Comment by mingtsay** [comment]
> Does this command hang? `busctl --user call org.freedesktop.portal.Desktop /org/freedesktop/portal/desktop org.freedesktop.portal.Settings Read ss org.freedesktop.appearance color-scheme`

No, but it failed to run instead.

```zsh
mt@ming-debian [00時08分01秒] [~] 
-> % busctl --user call org.freedesktop.portal.Desktop /org/freedesktop/portal/desktop org.freedesktop.portal.Settings Read ss org.freedesktop.appearance color-scheme
Call failed: Requested setting not found
mt@ming-debian [00時08分10秒] [~] 
-> % sudo busctl --user call org.freedesktop.portal.Desktop /org/freedesktop/portal/desktop org.freedesktop.portal.Settings Read ss org.freedesktop.appearance color-scheme
Failed to set bus address: $DBUS_SESSION_BUS_ADDRESS and $XDG_RUNTIME_DIR not defined (consider using --machine=<user>@.host --user to connect to bus of other user)
mt@ming-debian [00時08分13秒] [~] 
-> % 
```

**Comment by ilya-fedin** [comment]
The first answer is good one. It's ok to not to have the setting.

**Comment by ilya-fedin** [comment]
Does it hang after the window is open or before?

**Comment by mingtsay** [comment]
> Does it hang after the window is open or before?

It hangs after the window is showed up.
No actions when the "Start Messaging" button is hovered and/or clicked.

Before I deleted the folder `.local/TelegramDesktop/`, it hangs when the window shows the main layout without loading any message and Telegram folders.

Also, I need to run `killall Telegram` to kill it. It cannot be closed by mouse clicks.

**Comment by mingtsay** [comment]
> Are you sure it's happening since 4.7.1, not since 4.8.0? There's no change in 4.7.1 that change the behavior when there's no active settings.

You're right. I can confirm that v4.7.1 works normally but 4.8.0 don't.
Let me build it from source to try to debug it.

**Comment by ilya-fedin** [comment]
If this is since 4.8.0, I have an idea: try to start tdesktop with `ALSOFT_DRIVERS=null`

**Comment by mingtsay** [comment]
I'm still building the Telegram for debug. It may takes some time.

> If this is since 4.8.0, I have an idea: try to start tdesktop with `ALSOFT_DRIVERS=null`

Using `ALSOFT_DRIVERS=null` causes the following output.
`錯誤的檔案敘述項` is the translation from the string `Bad file descriptor`.

```zsh
mt@ming-debian [07時33分24秒] [~] 
-> % ALSOFT_DRIVERS=null ~/apps/Telegram/Telegram
QPainter::begin: Paint device returned engine == 0, type: 2
QWidget::render: Cannot render with an inactive painter
handle_write: 錯誤的檔案敘述項
hime_im_client_focus_in error: 錯誤的檔案敘述項
handle_write: 錯誤的檔案敘述項
hime_im_client_focus_in error: 錯誤的檔案敘述項
handle_write: 錯誤的檔案敘述項
hime_im_client_focus_in error: 錯誤的檔案敘述項
handle_write: 錯誤的檔案敘述項
hime_im_client_focus_in error: 錯誤的檔案敘述項
handle_write: 錯誤的檔案敘述項
hime_im_client_focus_in error: 錯誤的檔案敘述項
handle_write: 錯誤的檔案敘述項
hime_im_client_focus_in error: 錯誤的檔案敘述項
handle_write: 錯誤的檔案敘述項
hime_im_client_focus_in error: 錯誤的檔案敘述項
handle_write: 錯誤的檔案敘述項
hime_im_client_focus_in error: 錯誤的檔案敘述項
handle_write: 錯誤的檔案敘述項
hime_im_client_focus_in error: 錯誤的檔案敘述項
handle_write: 錯誤的檔案敘述項
hime_im_client_focus_in error: 錯誤的檔案敘述項
^C
mt@ming-debian [07時34分00秒] [~] 
-> % 
```

I `Ctrl-C` since it still hangs and keep printing the `Bad file descriptor` messages.

**Comment by ilya-fedin** [comment]
> Using `ALSOFT_DRIVERS=null` causes the following output.

Are you sure it's it? Maybe you just opened Telegram from terminal first time?

**Comment by mingtsay** [comment]
> Try to run tdesktop as `env -u QT_IM_MODULE ~/apps/Telegram/Telegram`

I run this command and Telegram never hangs again.
So the `QT_IM_MODULE` causes the problem.
Unset the `QT_IM_MODULE` variable solve the issue.

**Comment by ilya-fedin** [comment]
It's HIME what causes the problem... The binary has Qt built without Xlib support as a part of the work of making X11 dependencies optional in runtime for the binary. Sadly HIME seem to use Xlib, so we have now to choose between removing HIME support or reverting the change and delay the work even more.

**Comment by mingtsay** [comment]
> The binary has Qt built without Xlib support as a part of the work of making X11 dependencies optional in runtime for the binary.

There is a repo that has been archived: https://github.com/telegramdesktop/hime
Maybe we can let users to choose to support Xlib or not by any chance?
But maintain two versions of Telegram Desktop binary seems not to be good.

I think HIME should drop the Xlib dependency, or just tell user to use ether `tdesktop v4.7.1` or other IME.

**Comment by ilya-fedin** [comment]
> Maybe we can let users to choose to support Xlib or not by any chance?

If providing multiple builds wasn't a problem then I guess we would do separate X11 and Wayland builds instead, each one with required dependencies. But the goal is to have maximum cross-distro support with one binary.

> I think HIME should drop the Xlib dependency

I'm really skeptical it can do that, it has Xlib structs in its public API (e.g. the Display one).

**Comment by ilya-fedin** [comment]
I really don't understand why people use things like hime and nimf when there are well-maintained fcitx and ibus?

**Comment by mingtsay** [comment]
> I really don't understand why people use things like hime and nimf when there are well-maintained fcitx and ibus?

For Bopomofo, fcitx and ibus provide only Chewing (新酷音).
hime is forked from gcin, and it provides both Chewing and gcin(詞音).
gcin is much more similar to Microsoft Bopomofo IME on Windows than Chewing.
Also, HIME is 姫(ひめ) in Japanese, which means princess.

Anyway, I am planing to switch to fcitx when I found the IME which is similar to hime that running under fcitx.

**Comment by ilya-fedin** [comment]
Isn't this what you want?
https://packages.debian.org/bullseye/ibus-zhuyin

**Comment by mingtsay** [comment]
> Isn't this what you want? https://packages.debian.org/bullseye/ibus-zhuyin https://packages.debian.org/bullseye/ibus-libzhuyin

I now switched to `ibus-libzhuyin`. I have trouble installing `fcitx5-zhuyin`.
Thank you for the time to invest the problem and find solutions for me.

> If providing multiple builds wasn't a problem then I guess we would do separate X11 and Wayland builds instead, each one with required dependencies. But the goal is to have maximum cross-distro support with one binary.

Look forward to the best support with one binary.

**Comment by mingtsay** [comment]
I can confirm that this issue is solved by updating to v4.8.3 FYI
